README.md

```diff
@@ -1,77 +1,373 @@
-This can also be used as a great refresher for grads and newbies interested alike.
-While, we don't expect you to be experts, but we do expect you to be **familiar**.
-There is a lot of content to consume here, get used to the fire hose! 👨🏻‍🚒
+Welcome to the official Vets Who Code Prework! Here you'll find the tools, work, and accounts to prepare you for Vets Who Code's prior coursework.  We recommend you complete each section as they build upon one another. The prework should take a total of eight hours to complete.
 
-<h2 id="web-presence">Web Presence</h2>
 
-You should have a professional presence on all of the below platforms, as well as a professional email account. A [personal brand](http://blog.thefirehoseproject.com/posts/personal-branding-software-developer/) is really important.
+ ### Accounts
 
-- [Twitter](https://twitter.com)
-- [Codepen](https://codepen.io)
-- [GitHub](https://github.com)
-- [Dev.to](https://dev.to/)
-- [LinkedIn Veteran](https://linkedinforgood.linkedin.com/programs/veterans/premiumform)
+Here are the accounts you will need to store your projects, solve problems, prototype, network, write, and be inspired by others' work. You'll also be able to see some fantastic projects from others.
 
-Another important aspect of communicating your brand on the Internet as a developer is being able to use Markdown formatting. Markdown is a way to write plain text documents that get formatted into html for display on the web. This guide you're currently reading is written in Markdown. Markdown is extremely popular in 2018 and as part of the prework, you're going to take some time to understand it. <br />[Markdown Tutorial](https://www.markdowntutorial.com/)  
-[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
 
-<h2 id="command-line">Know Your Tool</h2>
+#### [LinkedIn For Veterans](https://socialimpact.linkedin.com/programs/veterans/premiumform)
 
-Being able to code well requires you to be able to use your machine well. We're going to start with setting up your machine and then move into learning a few tools that are staples to web development. If you're on Windows your setup instructions are [here](https://github.com/Vets-Who-Code/prework/blob/master/Windows-Tooling.md), MacOS users look [here](https://github.com/Vets-Who-Code/prework/blob/master/MacOS-Tooling.md).
+As a member of the U.S. military community, you can receive one year of LinkedIn Premium through SheerID's collaboration with LinkedIn. This service offers unlimited access to more than 10,000 courses through our LinkedIn Learning platform as a part of the program.
 
-### Code Editor
+LinkedIn Learning is also pleased to offer eligible military spouses one year of access to LinkedIn Premium. As part of the program, you'll also gain access to more than 17,000 courses through LinkedIn Learning to help you manage your career and nurture your professional development.
```

Comment:
> spouses one year of access to LinkedIn Premium. [...] more than 17,000 courses

Why do spouses get access to 17,000 courses while veterans get access to 10,000 courses. Could we combine military community and their spouses into a single sentence?

Reply:
The program is about LinkedIn Premium's Social impact program. They also haven't updated their language. It can be updated and merged.